x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/map_filters_to_kuery_node.ts

```diff
@@ -63,6 +65,19 @@ export const mapFiltersToKueryNode = ({
     );
   }
 
+  if (ruleParamsFilter && Object.keys(ruleParamsFilter).length) {
+    filterKueryNode.push(
+      nodeBuilder.or(
+        Object.keys(ruleParamsFilter).map((ruleParam) =>
+          nodeBuilder.is(
+            `alert.attributes.params.${ruleParam}`,
+            String(ruleParamsFilter[ruleParam])
+          )
+        )
+      )
+    );
+  }
```

Comment:
@kdelemme this adds the `or` clause. I'll make it `and` instead

Reply:
Alright :)